Sources/SwiftBundler/Extensions/Process.swift

```diff
@@ -7,12 +7,12 @@ import Foundation
 /// All processes that have been created using `Process.create(_:arguments:directory:pipe:)`.
 ///
 /// If the program is killed, all processes in this array are terminated before the program exits.
-var processes: [Process] = []
+public var processes: [Process] = []
 
 #if os(Linux)
   /// The PIDs of all AppImage processes started manually (due to the weird
   /// workaround required).
-  var appImagePIDs: [pid_t] = []
+  public var appImagePIDs: [pid_t] = []
```

Comment:
`appImagePIDs` is now a free, public variable "polluting" the library user namespace. I'd prefer to encapsulate this in a enum e.g. `Process`. What do you think @stackotter?

Reply:
Same as `processes` (although this particular global will probably disappear pretty soon; I have a feeling that Greg's changes will have fixed the `Process` issue I was facing when trying to run appimages)

Reply:
I've decided to go a different route and just add a public function `Process.killAllRunningProcessesOnExit()`. This gets rid of the need to expose any of this and refactoring can be done whenever. 
Changed with 216e1a8cb4a1442477c43ce4db7b51ed6883cf78